lib/fs.js

```diff
@@ -837,7 +840,11 @@ function ftruncate(fd, len = 0, callback) {
 function ftruncateSync(fd, len = 0) {
   validateInt32(fd, 'fd', 0);
   validateInteger(len, 'len');
-  len = MathMax(0, len);
+
+  if (len < 0) {
+    throw new ERR_INVALID_ARG_VALUE(len);
```

Comment:
```suggestion
    throw new ERR_INVALID_ARG_VALUE('len', len);
```

Reply:
I think that with this change there will not be a better developer experience, it will be the same 🤔
Isn't it?

Reply:
Ops! 😅